### ISSUE TITLE
how to train model on my custom  data?

### ISSUE THREAD

**Issue by BaofengZan** [sair-lab/AirLine#7]

**Comment by Lx017** [comment]
Hi! Although we did not include our training script it will be just very similar to edge detection training, which is easy to code, as edge-to-line conversion is not related to training in this work. We did nothing else but to use a weighted BCE pixel-by-pixel loss as described in the paper. if you have other questions I am happy to answer!

**Comment by BaofengZan** [comment]
Thank you for your reply.  Is there any relevant repository  for reference?

**Comment by Lx017** [comment]
i think unet++ relevant should work.
